server/config/config.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
You may consider using environment variable `.env` to store sensitive database information rather than exposing them in your `config.json` file